test/helpers.ts

```diff
@@ -7,9 +7,13 @@ export const gitLfsVersion = '2.13.2'
 
 const temp = require('temp').track()
 
-export async function initialize(repositoryName: string): Promise<string> {
+export async function initialize(repositoryName: string, defaultBranch?: string): Promise<string> {
```

Comment:
Reason I didn't just simply make it use a default branch of 'main' all the time.. is it broke a few other tests and it was not obvious as to why and did not want to spend the time right now figuring that out/being concerned with unintended impacts (just wanted to fix the 1 failing test.. not break others 😄 )